packages/web/components/place-limit-tool/index.tsx

```diff
@@ -12,40 +11,55 @@ import {
   useState,
 } from "react";
 
-import { Icon } from "~/components/assets";
 import { TokenSelectLimit } from "~/components/control/token-select-limit";
 import { LimitInput } from "~/components/input/limit-input";
 import { LimitPriceSelector } from "~/components/place-limit-tool/limit-price-selector";
 import { LimitTradeDetails } from "~/components/place-limit-tool/limit-trade-details";
 import { TRADE_TYPES } from "~/components/swap-tool/order-type-selector";
 import { TradeDetails } from "~/components/swap-tool/trade-details";
 import { Button } from "~/components/ui/button";
-import { useTranslation, useWalletSelect } from "~/hooks";
+import { useDisclosure, useTranslation, useWalletSelect } from "~/hooks";
 import { OrderDirection, usePlaceLimit } from "~/hooks/limit-orders";
-import { useOrderbookAllActiveOrders } from "~/hooks/limit-orders/use-orderbook";
+import { AddFundsModal } from "~/modals/add-funds";
 import { ReviewLimitOrderModal } from "~/modals/review-limit-order";
 import { useStore } from "~/stores";
 
 export interface PlaceLimitToolProps {
   orderDirection: OrderDirection;
+  fromAssetsPage?: boolean;
 }
 
 const WHALE_MESSAGE_THRESHOLD = 100;
 
 export const PlaceLimitTool: FunctionComponent<PlaceLimitToolProps> = observer(
-  () => {
+  ({ fromAssetsPage }) => {
```

Comment:
Might be a bit of redundancy here with https://github.com/osmosis-labs/osmosis-frontend/pull/3558 passing the `page` prop.